director/director_api.go

```diff
@@ -33,3 +49,78 @@ func ListNamespacesFromOrigins() []NamespaceAd {
 	}
 	return namespaces
 }
+
+func LoadDirectorPublicKey() (*jwk.Key, error) {
+	directorUrlStr := param.Federation_DirectorUrl.GetString()
+	if len(directorUrlStr) == 0 {
+		return nil, errors.Errorf("Director URL is unset; Can't load director's public key")
+	}
+	log.Debugln("Director URL:", directorUrlStr)
+	directorUrl, err := url.Parse(directorUrlStr)
+	if err != nil {
+		return nil, errors.Wrap(err, fmt.Sprintln("Invalid director URL:", directorUrlStr))
+	}
+	directorUrl.Scheme = "https"
+
+	discoveryUrl, _ := url.Parse(directorUrl.String())
+	discoveryUrl.Path = path.Join(".well-known/pelican-configuration", discoveryUrl.Path)
+
+	tr := config.GetTransport()
+	client := &http.Client{Transport: tr, Timeout: time.Second * 5}
```

Comment:
Any particular reason we set the timeout here instead of using the default? It's fine if there is a reason, but if that's the case can you explain with a quick comment?

Reply:
I referred to the similar behavior of [DiscoverFederation() function](https://github.com/PelicanPlatform/pelican/blob/e7fc4aa86870acd444e3f43b7388bba6a99748db/config/config.go#L169) in config.go. I agree we should just use default setting here.